Order/Order.Domain/Interfaces/IUserRepository.cs

```diff
@@ -4,6 +4,6 @@ namespace Order.Domain.Interfaces;
 
 public interface IUserRepository: IRepository<User, long>
 {
-	bool DoesItExist(long id);
-	bool DoesItExist(User user);	
+	Task<bool> DoesItExistAsync(long id);
+	Task<bool> DoesItExistAsync(User user);	
```

Comment:
Methods returning boolean values should start with 'Is'. Writing 'DoesIt...' is incorrect. It's better to replace the beginning of the method name with 'IsUser...'